src/main/webapp/app/exercises/shared/exercise/exercise.utils.ts

```diff
@@ -196,10 +198,22 @@ export const participationStatus = (exercise: Exercise, testRun?: boolean): Part
     return ParticipationStatus.INACTIVE;
 };
 
-export const isStartExerciseAvailable = (exercise: ProgrammingExercise): boolean => {
-    return !exercise.dueDate || dayjs().isBefore(exercise.dueDate);
+/**
+ * Determines if the exercise can be started, this is the case if:
+ * - It is after the start date or the participant is at least a tutor
+ * - In case of a programming exercise it is not before the due date
+ * @param exercise the exercise that should be started
+ */
+export const isStartExerciseAvailable = (exercise: Exercise): boolean => {
```

Comment:
```suggestion
export const canStartExercise = (exercise: Exercise): boolean => {
```

```suggestion
export const allowExerciseStart = (exercise: Exercise): boolean => {
```

With the current wording, I would expect a check whether something is present in the entity

Reply:
If you agree, please fix similar occurrences in this service